app/Account.php

```diff
@@ -580,6 +603,76 @@ public function replaceGender(Gender $genderToDelete, Gender $genderToReplaceWit
         return true;
     }
 
+    /**
+     * Get if any account exists on the database.
+     *
+     * @return bool
+     */
+    public static function hasAny()
+    {
+        return DB::table('accounts')->count() > 0;
+    }
+
+    /**
+     * Create a new account and associate a new User.
+     *
+     * @param string $first_name
+     * @param string $last_name
+     * @param string $email
+     * @param string $password
+     * @return this
+     */
+    public static function createDefault($first_name, $last_name, $email, $password)
+    {
+        // create new account
+        $account = new self;
+        $account->api_key = str_random(30);
+        $account->created_at = Carbon::now();
+        $account->save();
+
+        $account->populateDefaultFields($account);
+
+        // create the first user for this account
+        User::createDefault($account->id, $first_name, $last_name, $email, $password);
+
+        return $account;
+    }
+
+    /**
+     * Populates all the default column that should be there when a new account
+     * is created or reset.
+     */
+    public static function populateDefaultFields($account)
+    {
+        $account->populateContactFieldTypeTable();
+        $account->populateDefaultGendersTable();
+        $account->populateDefaultReminderRulesTable();
+        $account->populateRelationshipTypeGroupsTable();
+        $account->populateRelationshipTypesTable();
+    }
+
+    /**
+     * Gets the RelationshipType object matching the given type.
+     *
+     * @param  string $relationshipTypeName
+     * @return RelationshipType
+     */
+    public function getRelationshipTypeByType(string $relationshipTypeName)
+    {
+        return $this->relationshipTypes->where('name', $relationshipTypeName)->first();
```

Comment:
add `->get()` to return the object, and not the `Query\Builder` instance

Reply:
`get` would return a collection, I need a single item.

Reply:
ok ok